### DISCUSSION TITLE
Read Modify Write Capabilities

### DISCUSSION THREAD

**Discussion by koculu** [microsoft/FASTER#671] [Ideas]
Hi,
I would like to cancel writes in certain conditions. Is there a Cancel Write option in RMW functions?
If not, would it be possible to add such a capability?
Thank you.

**Comment by badrishc** [comment]
This feature is actually part of our v2 preview, where IFunctions API takes a `RmwInfo` parameter, that has a `Cancel` action among others. You have to return `false` from the function in order for FASTER to perform the requested action. This is an experimental prototype at the moment, though.
